lib/ipam/ipam_block.go

```diff
@@ -32,15 +32,15 @@ import (
 
 // windwowsReservedHandle is the handle used to reserve addresses required for Windows
 // networking so that workloads do not get assigned these addresses.
-const windowsReservedHandle = "windows-reserved-IPAM-handle"
+const WindowsReservedHandle = "windows-reserved-IPAM-handle"
```

Comment:
might have spoke about this before but, I think this will be converted to lowercase when we write the Handle object to the k8s API since k8s resource names must be lowercase.

I think we should have it lowercase here as well so that all references to the handle name are identical.

Reply:
Good point. I did not see any issue of upper case but it is a good idea to make it lower case.